### ISSUE TITLE
iOS 7

### ISSUE THREAD

**Issue by bludginozzie** [onemightyroar/YRDropdownView#18]
I recently upgrade a project to iOS7 only to discover that the YRDropdownView appears behind the title bar.  Do I need change my code in iOS7?

![screen shot 2014-06-17 at 6 05 18 am](https://cloud.githubusercontent.com/assets/7906257/3292667/0bda5980-f592-11e3-94f9-71701dff35e6.png)

Thanks,
-Mark

**Comment by perry-mitchell** [comment]
@bludginozzie Did you get it working? From first glance, this looks as though it's because you haven't specified edgesForExtendedLayout. Think about it, in iOS 7 and 8, the views can go under the nav bars, so when you run code like:

```
[YRDropdownView showDropdownInView:self.view
                             title:@"Warning"
                            detail:@"Danger Will Robinson. You cannot do that."];
```

with the `showDropdownInView` method, the view is actually underneath the bar (and thus the dropdown will show there too.

I'm about to try this in an iOS 8 application, so here's hoping it works fine.

**Comment by bludginozzie** [comment]
Yep, I ended up changing this property however I would have preferred to keep the property and have YRDropdown smart enough to detect the layout and adjust accordingly.  I was reluctant to change the source in the event of upgrades.  Would be very interested to see how iOS8 goes.  I was using iOS7.
